cfme/login.py

```diff
@@ -7,77 +7,114 @@
 """
 from __future__ import absolute_import
 
+import time
 from selenium.webdriver.common.keys import Keys
-from selenium.common.exceptions import NoSuchElementException, ElementNotVisibleException
 
 import cfme.fixtures.pytest_selenium as sel
-import cfme.web_ui.flash as flash
-from cfme import dashboard, Credential
-from cfme.configure.access_control import User
-from cfme.web_ui import Region, Form, fill, Input
-from utils import conf, version
-from utils.browser import ensure_browser_open, quit
+from cfme import Credential
+from utils import conf
 from utils.log import logger
 from fixtures.pytest_store import store
 
+from widgetastic.widget import Text, View
+from widgetastic_patternfly import Button, Input, FlashMessages
+
+from . import BaseLoggedInPage
+
+
+class LoginPage(View):
```

Comment:
I am still not completely sure how that will look together with your changes ...